File: app/src/main/java/com/proxy/shadowsocksr/items/GlobalProfile.java
package com.proxy.shadowsocksr.items;

import java.io.Serializable;

public class GlobalProfile implements Serializable
{
    public String route = "bypass-lan";
    public boolean ipv6Route = false;
    public boolean globalProxy = true;
    public boolean dnsForward = true;
    public boolean autoConnect = false;
}

File: app/src/main/java/com/proxy/shadowsocksr/items/GlobalProfile.kt
package com.proxy.shadowsocksr.items

import java.io.Serializable

class GlobalProfile : Serializable
{
    var route = "bypass-lan"
    var ipv6Route = false
    var globalProxy = true
    var vpnMode = true
    var dnsForward = true
    var autoConnect = false
}
